packages/openapi3/src/cli/actions/convert/utils/get-scope-and-name.ts

```diff
@@ -5,7 +5,7 @@ export function getScopeAndName(originalName: string): ScopeAndName {
   const path = originalName.split(".");
   const name = path.pop()!;
 
-  return { scope: path.map(printIdentifier), name: printIdentifier(name), rawName: name };
+  return { scope: path.map((x) => printIdentifier(x)), name: printIdentifier(name), rawName: name };
```

Comment:
I'm surprised this change was needed.

Reply:
yeah I don't get why tbh, typescript not happy its not a single arg config